src/index.js

```diff
@@ -0,0 +1,13 @@
+import './index.css';
+import addHtml from './modules/addHtml.js';
+import btnAdd from './modules/buttonAdd.js';
+import refreshBtn from './modules/refresh.js';
+import { clearFunction } from './modules/check.js';
+
+const buttonAdd = document.getElementById('add-button');
```

Comment:
I think it is best practice if inside the brackets you used ('#add-button') to refer to an Id just like the way we use ('.add-button') for a class.